src/components/PlaceholderContainer/PlaceholderContainer.tsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+
+import {block} from '../utils/cn';
+
+import {Action, PlaceholderContainerAction} from './PlaceholderContainerAction';
+
+import './PlaceholderContainer.scss';
+
+const ImageSize = {
+    s: 100,
+    m: 150,
+    l: 230,
+    promo: 230,
+};
+
+const Direction = {
+    Row: 'row',
+    Column: 'column',
+} as const;
+const Align = {
+    Left: 'left',
+    Center: 'center',
+} as const;
+
+type Size = 's' | 'm' | 'l' | 'promo';
+
+type PlaceholderContainerImageNodeProps = NonNullable<React.ReactNode>;
+
+type PlaceholderContainerImageSettingsProps = {
+    url: string;
+    alt?: string;
+    size?: number;
```

Comment:
Not sure if we need this prop, we already have image constraints. In fact, image from url can be rectangle shape or smaller size than image contanier

Reply:
That's a good point. I removed this prop.